sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpEventBatchTests.cs

```diff
@@ -9,10 +9,10 @@
 using Azure.Messaging.EventHubs.Core;
 using Azure.Messaging.EventHubs.Producer;
 using Microsoft.Azure.Amqp;
-using Microsoft.Azure.Amqp.Framing;
 using Moq;
 using NUnit.Framework;
 using NUnit.Framework.Constraints;
+using FramingData = Microsoft.Azure.Amqp.Framing.Data;
```

Comment:
There was a naming clash, I believe with something in Experimental? I'd have to double check.

Reply:
I'll be honest.   This makes me uncomfortable.    If we can, I'd like to revise/revert.

Reply:
After looking into it and discussion, the clash was with `Azure.Data` namespace from `Azure.Data.SchemaRegistry`. We decided the best option was to keep the alias since partial qualification (`Framing.Data`) won't work here as we're not inside of a `Microsoft.Azure.Amqp` namespace.